control_plane/src/local_env.rs

```diff
@@ -911,6 +938,15 @@ impl LocalEnv {
         // create endpoints dir
         fs::create_dir_all(env.endpoints_path())?;
 
+        // create storage controller dir
+        fs::create_dir_all(env.storage_broker_data_dir())?;
+        if generate_local_ssl_certs {
+            env.generate_ssl_cert(
+                &env.storage_broker_data_dir().join("server.crt"),
+                &env.storage_broker_data_dir().join("server.key"),
+            )?;
+        }
```

Comment:
```suggestion
        // create storage broker dir
        fs::create_dir_all(env.storage_broker_data_dir())?;
        if generate_local_ssl_certs {
            env.generate_ssl_cert(
                &env.storage_broker_data_dir().join("server.crt"),
                &env.storage_broker_data_dir().join("server.key"),
            )?;
        }
```

Reply:
Also, seems like it's time to introduce the `StorageBroker::from_env().init()` pattern that we have for all the other entities, and move the code in this quote there. Can you do that in this PR?

Reply:
Done